datanode/sqlstore/migrations/0096_add_deposit_rejected_state.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+
+ALTER TYPE deposit_status ADD VALUE 'STATUS_DUPLICATE_REJECTED';
```

Comment:
```suggestion
ALTER TYPE deposit_status ADD VALUE IF NOT EXISTS 'STATUS_DUPLICATE_REJECTED';
```

Otherwise it could cause issue with network history because there's no down migration.